app/controllers/application_controller.rb

```diff
@@ -240,6 +240,16 @@ def render_full_width(template, **opts)
     render template, **opts, layout: 'base'
   end
 
+  def user_needs_sign_up_completed_page?
+    issuer = sp_session[:issuer]
+    return false unless issuer
+    !user_has_identity_for_issuer?(issuer)
+  end
+
+  def user_has_identity_for_issuer?(issuer)
```

Comment:
This change means that IAL2 users won't have to provide explicit consent to share new attributes with an SP if an SP requests an attribute that it has not requested before.

Reply:
Switched this to IAL1 only to leave existing IAL2 behavior alone